cas-compute/src/symbolic/mod.rs

```diff
@@ -76,6 +76,7 @@
 
 pub mod expr;
 pub mod simplify;
+pub mod derivative;
 pub mod step_collector;
```

Comment:
```suggestion
pub mod derivative;
pub mod expr;
pub mod simplify;
pub mod step_collector;
```

Placing modules in alphabetical order.